.gitignore

```diff
@@ -76,3 +76,4 @@ __pycache__
 
 # Allow to exist but don't include it in the repo
 user_song_list.h
+*.orig
```

Comment:
Please revert this.  

If you need to ignore it still, you can do this in your global/system config, rather than here.